### DISCUSSION TITLE
Forge

### DISCUSSION THREAD

**Discussion by ghost** [EssentialsX/Essentials#5240] [General]
Hello, Does Essentials work with Forge.

**Comment by pop4959** [comment]
No

**Comment by ghost** [comment]
What does it work with then

**Reply by pop4959** [reply]
It's a Bukkit plugin, you can use it on CraftBukkit, Spigot, or Paper servers.

**Reply by ghost** [reply]
Okay

**Comment by ghost** [comment]
The reason I'm asking is that I joined a Forge Server recently, and it had the same commands as Essentials.

**Reply by pop4959** [reply]
There are probably similar mods for Forge.

**Reply by ghost** [reply]
Okay, I will look into it more. Thanks for the help!
